### ISSUE TITLE
Unable to Fetch MAM stream, if repeatedly posting a transaction on the same address/root

### ISSUE THREAD

**Issue by JelleMillenaar** [iotaledger-archive/mam.client.js#16]
The fetch function fails to complete with the following error: 
thread '<unnamed>' panicked at 'index 3146175440 out of range for slice of length 19683', libcore/slice/mod.rs:785:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
failed to parse:  5273224

The problem is that I didn't follow the MAM consensus in posting a single transaction per address. After repeatedly using the first address generated from the seed, the data becomes too long for the fetch function to handle. This error should probably be caught.

**Comment by sobolev-alexey** [comment]
Can you post your code, so we can reproduce the issue exactly as you see it?

**Comment by Core-pl** [comment]
I had the same issue. Idea is following:
1. Generate seed
2. Generate root from that seed
3. Send message to that root
4. Send another message to the same root (so now we have 2 messages on the same root)
5. Try to read that root - it will crash, even if you wrap it in try/catch, because issue is not handled in rust code

I did not check if this issue still exists (last time I checked it about 3 months ago). At that time my only work-around was to be super-cautious about not allowing messaging the channel in parallel (do not allow to send next message till previous is attached). The worst thing is that if by mistake somebody re-use a root, it will block the channel.

Alex, if the above explanation is not enough, let me know, I will elaborate.
